proto/resource_manager.proto

```diff
@@ -196,6 +196,7 @@ enum RunawayAction {
 enum RunawayWatchType {
   Exact = 0;
   Similar = 1;
+  Plan = 3;
```

Comment:
If I understand correctly, in the current impl, Similar actually means Plan, so after this change Similar will mean SQL digest, right? So this is a incompatible change

Reply:
Yes, we should also modify the docs